radixdlt-cli/build.gradle

```diff
@@ -42,11 +42,12 @@ compileJava {
     options.compilerArgs += ["-Aproject=${project.group}/${project.name}"]
 }
 dependencies {
-    compile 'org.codehaus.groovy:groovy-all:2.4.15'
-    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.2'
-    compile group: 'commons-cli', name: 'commons-cli', version: '1.4'
-    compile 'info.picocli:picocli:4.2.0'
-    annotationProcessor 'info.picocli:picocli-codegen:4.2.0'
+    compile 'org.codehaus.groovy:groovy-all:3.0.5'
+    compile 'com.google.code.gson:gson:2.8.2'
+    compile 'commons-cli:commons-cli:1.4'
+    compile 'info.picocli:picocli:4.4.0'
+    compile('com.google.guava:guava:27.1-jre') { force = true }
```

Comment:
Why do we need a "force" here?  Would prefer if we didn't.

Reply:
after updating groovy there were conflicts so I was ought to force version to resolve them

Reply:
I'll try to figure out how to resolve conflicts in less intrusive way